test/unit/modules/cache.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+var node = require('./../../node.js');
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var async = require('async');
+var sinon = require('sinon');
+var modulesLoader = require('../../common/initModule').modulesLoader;
+var Cache = require('../../../modules/cache.js');
+
+describe('cache module', function () {
+
+	var cache;
+
+	before(function (done) {
+		modulesLoader.initCache(function (err, __cache) {
+			cache = __cache;
+			expect(err).to.not.exist;
+			expect(__cache).to.be.an('object');
+			cache = __cache;
+			return done();
+		});
+	});
+
+	afterEach(function (done) {
+		cache.flushDb(function (err, status) {
+			expect(err).to.not.exist;
+			expect(status).to.equal('OK');
+			done(err, status);
+		});
+
+	});
+
+	function itIfCacheEnabled (name, cb) {
```

Comment:
Because we want to allow people to run lisk application without cache. And if the wish to do so, they don't need to run cache tests. Plus, if I just set `config.cacheEnabled = true` and don't have redis server installed or haven't specified cache parameters, these tests will fail. Which is is the expected behaviour.